src/Mod/Draft/DraftGeomUtils.py

```diff
@@ -65,7 +65,9 @@
                                    getBoundaryAngles)
 
 # Needs general functions
-from draftgeoutils.geometry import (findPerpendicular,
+from draftgeoutils.geometry import (isStraightLine,
+                                    areCoplanar,
```

Comment:
If you add new functions make sure they use `snake_case` style. This is in the Draft coding conventions. I know many functions use the `camelCase` style, but this is legacy code that was written in 2010 or so, before we established conventions. Eventually the plan is to migrate the older functions to the new style, and use some compatibility layer, like
```py
def findPerpendicular():
    return find_perpendicular()
```